pkg/beaconproxy/mongodb.go

```diff
@@ -63,7 +63,7 @@ func (r *repo) CreateIndexes() error {
 }
 
```

Comment:
Please change the indexes line above on lines 48-54 to 
```go
indexes := []mgo.Index{
	{Key: []string{"-score"}},
	{Key: []string{"src", "fqdn", "src_network_uuid"}},
	{Key: []string{"src", "src_network_uuid"}},
	{Key: []string{"fqdn"}},
//	{Key: []string{"proxy.ip", "proxy.network_uuid"}}, this one's up to you if you think we might need it in the future
	{Key: []string{"-connection_count"}},
}
```

This incorporates the original index fix that was included in the first draft of #689.

Reply:
Changed the indexes to the ones you mentioned